observability/readme.md

```diff
@@ -11,12 +11,20 @@ Building observable systems enables development teams at CSE to measure how well
 
 ## Sections
 
-- [Pillars of Observability](#pillars-of-observability)
-- [Recommended Practices](#recommended-practices)
-- [Pitfalls to Avoid](#pitfalls-to-avoid)
-- [Recommended Tools and Approaches](#recommended-tools-and-approaches)
-- [Tools](#tools)
-- [Recipes](#recipes)
+- [Observability](#observability)
```

Comment:
Table of content does not have the latest from master

Reply:
hrm, great call-out. I'll make sure to do a rebase over latest.

Reply:
actually turned out my vs-code was set to auto-generate this on every save :), so that has been fixed.